c2corg_ui/static/js/map/map.js

```diff
@@ -451,7 +451,7 @@ app.MapController.prototype.createPointStyle_ = function(feature, resolution) {
 
```

Comment:
Yes it's a mistake ... I have do some test without sucess and commit too quickly. I'm not able to revert ... i have take time to found how do but ...
I have plan to do proper branch when this one work.